### PR TITLE
added vikaswakde.md

### PR DESCRIPTION
**Pull request by vikaswakde**

### CODE REVIEW

File: contributors/vikaswakde.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Name: Vikas Wakde (https://github.com/vikaswakde)
```
Comment (rupeshexe) [inline contributors/vikaswakde.md:1]: The name should be in the square brackets and there should not be any space between name and link.

>> [Your Name](https://github.com/<your_github_username>)

Comment (rupeshexe) [reply contributors/vikaswakde.md:1]: image

File: contributors/vikaswakde.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+[Twitter](https://twitter.com/vikaswakdeos) | [GitHub](https://github.com/vikaswakde) | [LinkedIn]([https://linkedin.com/in/username](https://www.linkedin.com/in/vikas-wakde-a7b1b6227/))
```
Comment (rupeshexe) [inline contributors/vikaswakde.md:3]: edit the linkedin url properly like github and twitter

**Comment by rupeshexe** [comment]
Or you can just copy this and enter usernames at last @vikaswakde 
```md
Name: [Your Name](https://github.com/<your_github_username>)

[Twitter](https://twitter.com/username) | [GitHub](https://github.com/username) | [LinkedIn](https://linkedin.com/in/username)

Description: <Anything you Like>.
```

File: contributors/vikaswakde.md (hunk 1 above)
Comment (vikaswakde) [reply contributors/vikaswakde.md]: corrected it! so sorry for the mistake.

File: contributors/vikaswakde.md (hunk 2 above)
Comment (vikaswakde) [reply contributors/vikaswakde.md]: I don't know how even I not cross-checked it, again sorry about that! 
Corrected it

**Comment by vikaswakde** [comment]
corrected all the mistakes and made a new PR

File: contributors/vikaswakde.md (hunk 2 above)
Comment (rupeshexe) [reply contributors/vikaswakde.md]: No need to say sorry, you had a great start!!

**Comment by rupeshexe** [comment]
Nice Work 🎉

**Comment by rupeshexe** [comment]
You can share this on your socials and tag CodeYacht and me!! 🚀

**Comment by vikaswakde** [comment]
> You can share this on your socials and tag CodeYacht and me!! rocket

Hey, how do I share it?

**Comment by vikaswakde** [comment]
Thank you for me the opportunity to contribute.

**Pull request merged**